### ISSUE TITLE
[Bug] Missing @luma.gl/core and @luma.gl/constants peer dependencies

### ISSUE THREAD

**Issue by NameFILIP** [visgl/deck.gl#6661]
### Description

The `@luma.gl/core` and `@luma.gl/constants` libraries are accessed in the code (Example: https://github.com/visgl/deck.gl/blob/master/modules/layers/src/arc-layer/arc-layer.js#L24) but they are not defined as either peer or regular dependencies in the `@deck.gl/layers` [package.json](https://github.com/visgl/deck.gl/blob/master/modules/layers/package.json)are not defined as either

### Flavors

- [ ] React
- [ ] Python/Jupyter notebook
- [ ] MapboxLayer
- [ ] GoogleMapsOverlay
- [ ] CartoLayer
- [ ] DeckLayer/DeckRenderer for ArcGIS

### Environment

The latest deck.gl master (v8.7.0-beta.5)

**Comment by Pessimistress** [comment]
Is this causing any issues? `@deck.gl/layers` has `@deck.gl/core` as peer dependency, which calls for all of the other dependencies.

**Comment by NameFILIP** [comment]
Not causing any issues in the behavior, but yarn (v2) displays warnings when installing the dependencies

**Comment by davidcalhoun** [comment]
Can confirm this on a similar setup using yarn 3 here.  We have a workaround in place at the moment to patch in a few dependencies like this in `.yarnrc.yml`:

```
packageExtensions:
    '@deck.gl/layers@*':
        dependencies:
            '@luma.gl/core': '*'
            ...
```

**Comment by Pessimistress** [comment]
This will be fixed in 8.8.0.
